docs/readme-sync/v4.0/deploy-as-a-microservice/040 - configure-optimizely-agent/010 - authorization.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Authorization"
+excerpt: ""
+slug: "authorization"
+hidden: false
+metadata: 
+  title: "Agent Authorization - Optimizely Full Stack"
+createdAt: "2020-03-11T20:58:11.777Z"
+updatedAt: "2020-03-31T19:44:52.119Z"
```

Comment:
same comment for all the documents, should we update the updatedAt date?

Reply:
done for all